File: tech-studio-projects/other-projects/enterprise-it-network-simulation/golang/enterprise-it-network-simulation/logger/logger.go
package logger

import (
	"log"
	"os"
)

// Init initializes the logger
func Init() {
	logFile, err := os.OpenFile("simulation.log", os.O_CREATE|os.O_RDWR|os.O_APPEND, 0666)
	if err != nil {
		log.Fatal("Error opening log file: ", err)
	}
	log.SetOutput(logFile)
	log.Println("Logger initialized.")
}

// LogInfo logs information level messages
func LogInfo(message string) {
	log.Println("[INFO] " + message)
}

// LogError logs error level messages
func LogError(message string) {
	log.Println("[ERROR] " + message)
}

File: tech-studio-projects/financial-industry/portfolio-management-simulation-platform/csharp/Core/Core/Utils/Logger.cs
using System;

namespace Core.Utils
{
    public static class Logger
    {
        public static void Log(string message)
        {
            Console.WriteLine($"[INFO] {DateTime.UtcNow}: {message}");
        }

        public static void LogError(string message)
        {
            Console.WriteLine($"[ERROR] {DateTime.UtcNow}: {message}");
        }
    }
}
